Events/eventModels.js

```diff
@@ -0,0 +1,15 @@
+var mongoose = require('mongoose');
+
```

Comment:
For consistency to how reports is done;
add this line `var Schema = mongoose.Schema;`